fixtures/downshift/src/hooks/useSelect/reducer.js

```diff
@@ -1,6 +1,6 @@
 import { getNextWrappingIndex, getNextNonDisabledIndex } from '../../utils';
-import { getHighlightedIndexOnOpen, getDefaultValue } from '../utils';
-import { getItemIndexByCharacterKey } from './utils';
+import { getHighlightedIndexOnOpen, getDefaultValue } from '../utils.js';
+import { getItemIndexByCharacterKey } from './utils.js';
```

Comment:
I'm curious why does the following not have the `.js` extension?
```js
import { getNextWrappingIndex, getNextNonDisabledIndex } from '../../utils';
```

Reply:
I think it only enforces it for files which it can find/actually exist.

So in this case, the file doesn't exist (since this file is just one of the fixtures files picked out of the downshift project, and other files in that project weren't included).

Reply:
Ah, interesting.